### PR TITLE
[Edit] JavaScript Variables

### LINKED ISSUE #5204: [Edit] JavaScript Variables
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

We would like to edit the `Variables` concept entry under JavaScript. The entry lives in `content/javascript/concepts/variables/variables.md`.

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

Please:
1) Add a `Codebyte Example` section that provides a codebyte example demonstrating the concept in use
2) Ensure that the codebyte example is different than all the other examples in the entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by smiosso**
### Description

- Added new codebite examples.

### Issue Solved
- Closes #5204 

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [ x] All writings are my own.
- [x ] My entry follows the Codecademy Docs style guide.
- [x ] My changes generate no new warnings.
- [x ] I have performed a self-review of my own writing and code.
- [x ] I have checked my entry and corrected any misspellings.
- [ x] I have made corresponding changes to the documentation if needed.
- [ x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Comment by smiosso** [comment]
Something went wrong, and I do not know why or how to fix it.

**Comment by Sriparno08** [comment]
No worries, @smiosso. The reviewers will fix it.

**Comment by smiosso** [comment]
Thank youuuuuu!!! ☺️

_________________________________
Samantha A. Colognese Tamiosso
+49 157 8510 6917



Em ter., 17 de set. de 2024 às 17:32, Sriparno Roy ***@***.***>
escreveu:

File: content/javascript/concepts/variables/variables.md [hunk 1]
```diff
@@ -36,6 +53,34 @@ let weight;
 const numberOfFingers = 20;
 ```
 
+```js
```
Comment (Sriparno08) [inline content/javascript/concepts/variables/variables.md:56]: Let's do the same changes as the previous one:

````suggestion
## Codebyte Example

The following codebyte example shows how to declare variables in JavaScript:

```codebyte/javascript
````

File: content/javascript/concepts/variables/variables.md [hunk 2]
```diff
@@ -45,6 +90,14 @@ let movieTitle = '300'; // String
 let audienceNumber = 300; // Number
 ```
 
+```js
+let magicNumber = "42";
+// The variable is a "string"
+
+magicNumber = 42;
+//Now the variable is a "number"
+```
```
Comment (Sriparno08) [inline content/javascript/concepts/variables/variables.md:99]: Let's remove this codebyte example as there's nothing to output here:

````suggestion
let audienceNumber = 300; // Number
```
````

**Review by Sriparno08 — CHANGES_REQUESTED** [review]
Hey @smiosso, thanks for contributing to Codecademy Docs!

I've listed all the modifications that you need to make in your entry below:

**Review by Sriparno08 — APPROVED** [review]
Hey @smiosso, this PR looks good for a second round of review!

**Review by mamtawardhani — APPROVED** [review]
Thank you for contributing to Codecademy Docs @smiosso 😄 

The entry is ready to be merged! 🚀

**Pull request merged**